apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -333,6 +333,19 @@ def from_anndata(
                 ) as data:
                     _maybe_set(x, "data", data, use_relative_uri=use_relative_uri)
 
+                for key in anndata.layers.keys():
+                    with create_from_matrix(
+                        X_kind,
+                        _util.uri_joinpath(measurement_X_uri, key),
+                        anndata.layers[key],
```

Comment:
actually, since you're doing this, `for key, layer in anndata.layers.items()` would probably be the best call, since then you can use `layer` here instead of indexing